pyrostest/ros_test.py

```diff
@@ -49,9 +47,8 @@ def __init__(self, topic, msg_type, queue_size):
     def send(self, value):
         """Sends data to be publoshed to the mocked topic.
         """
+        time.sleep(.5)
```

Comment:
this will wait half a second before each message we try to send...

isn't that rly bad